mgr/src/config.rs

```diff
@@ -175,6 +204,16 @@ pub struct Host {
 
     /// CockroachDB nodes to connect to
     pub cockroach_peers: Vec<CockroachPeer>,
+
+    /// Setup telegraf output auth for kuutamo monitor server
+    #[serde(skip_serializing)]
+    pub kmonitor_config: Option<KmonitorConfig>,
+
+    /// Has monitoring server or not
+    pub telegraf_has_monitoring: bool,
```

Comment:
`telegraf` is always on and open endpoint for others to query metrics.  Only `telegraf` has a remote monitoring setting, it will push the metrics to the server.  The remote server is `mimir` not `telegraf`.

Reply:
Oh I think the reason you don't want to change it is to keep the `telegraph` prefix, which is fine.